.circleci/config.yml

```diff
@@ -94,20 +94,41 @@ workflows:
   version: 2
   publish:
     jobs:
-      - checkout
+      - checkout:
+          filters:
+            branches:
+              ignore: master
       - build:
           requires:
             - checkout
-      - publish_versioned:
+      - manual_approval:
```

Comment:
manual_approval seems a bad name. 
The type already says is approval and we might even want several approvals.
e.g. `publish_approval`